infra/modules/azure_github_environment_bootstrap/README.md

```diff
@@ -73,11 +73,12 @@ The default branch name can be changed via the `default_branch_name` property.
 
 ## Extending the module for custom needs
 
-The module provides the basic configuration adhering to DX and Technology
-standards. However, it can be extended according to team needs.
+The module provides the basic configuration adhering to DX and Technology standards. However, it can be extended according to new needs. In fact, the module export all the ids and names of the resources that creates, so it is straightforward to add further resources.
```

Comment:
```suggestion
The module provides the basic configuration adhering to DX and Technology standards. However, it can be configured according to new needs. In fact, the module export all the ids and names of the resources that creates, so it is straightforward to add further resources.
```